<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora De Matrizes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de colunas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de linhas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de colunas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de linhas:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMatA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMatB">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicacao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTranspostaB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdicao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubtracao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTranspostaA">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicarBPorK">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz B * k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicarAPorK">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz A * k</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetSaida">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Triangular Superior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Inferior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Simétrica</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Identidade</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ortogonal</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>De Permutação</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>São Iguais</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>São Diferentes</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Matriz A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Matriz B</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEditQColunasMatA">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQLinhasMatA">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatrizA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Matriz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQColunasMatB">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQLinhasMatB">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatrizB">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Matriz</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz Resultante:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPotenciacaoA">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Potenciação A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPotenciacaoB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Potenciação B</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
